backend/src/govsg/middlewares/govsg-transactional.middleware.ts

```diff
@@ -149,6 +150,18 @@ export const InitGovsgTransactionalMiddleware =
           recipient: govsgTransactional.recipient,
           templateName: whatsappTemplateLabel,
           params: req.body.normalisedParams as NormalisedParam[],
+        }).catch((err) => {
+          if (err instanceof ApiInvalidRecipientError) {
+            govsgTransactional.set(
+              'status',
+              GovsgMessageStatus.InvalidRecipient
+            )
+            govsgTransactional.set('errorCode', err.httpStatusCode.toString())
+            govsgTransactional.set('errorDescription', err.errorCode)
```

Comment:
actually not super sure whether this is necessary, noticed that it's not done in the db function for campaign messages. but `INVALID_RECIPIENT` should be a type of error too imo.